lemur/common/schema.py

```diff
@@ -60,6 +60,8 @@ def wrap_with_envelope(self, data, many):
 class LemurInputSchema(LemurSchema):
     @pre_load(pass_many=True)
     def preprocess(self, data, many):
+        if isinstance(data, dict) and data.get('owner'):
+            data['owner'] = data['owner'].lower()
```

Comment:
Surely this code belongs in the schemas classes that actually have the `owner` field, not the base class?